### DISCUSSION TITLE
How to render an image from the web using image function in typst web app?

### DISCUSSION THREAD

**Discussion by klion26** [typst/typst#1219] [Q&A] [answered]
I want to know how to render an image from the web([1] for example) in typst? thanks.

I tried typing the command in typst web app like below, it shows "file not found (searched at https:/raw.githubusercontent.com/klion26/ImageRepo/master/20230516141825.png)".

searched the image function[2] and googled, but did not find the answer.

```
#figure(
  image(
    "https://raw.githubusercontent.com/klion26/ImageRepo/master/20230516141825.png",
    width:80%),
  caption: [tow-component-lsm],
)
```
[1] https://raw.githubusercontent.com/klion26/ImageRepo/master/20230516141825.png
[2] https://typst.app/docs/reference/visualize/image/#parameters--path

**Comment by Enivex** [comment, marked as answer]
Download the image first, upload it to the web app, and use its local relative path

**Comment by klion26** [comment]
@Enivex thanks for the reply, seems typst only support the local relative path now, and there is an rfc for external tools[1], will watch the RFC and close this issue.  

[1] https://github.com/typst/typst/issues/1056

**Comment by rvyhvn** [comment]
Would like it's supported!
